backend/src/modules/slide-objects/slide-objects.service.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import {
+  CreateSlideObjectDto,
+  UpdateSlideObjectDto,
+} from './slide-objects.dto';
+import {
+  SlideObject,
+  SlideObjectDocument,
+} from 'src/schemas/slide-object.schema';
+import { UsersService } from '../auth/users/users.service';
+import { SlidesService } from '../slides/slides.service';
+
+@Injectable()
+export class SlideObjectsService {
+  constructor(
+    @InjectModel(SlideObject.name)
+    private readonly slideObjectModel: Model<SlideObject>,
+    private readonly userService: UsersService,
+    private readonly slideService: SlidesService,
+  ) {}
+
+  async findAll(): Promise<SlideObjectDocument[]> {
+    return this.slideObjectModel.find().exec();
+  }
```

Comment:
Just a note here. This is just for test purposes and will never be needed on production. Endpoints like this should be deleted / disabled on production, especially that this endpoint is quite heavy.